polylogue/cli/init.py

```
@@ -0,0 +1,154 @@
+                return
+        else:
+            console.print(f"[yellow]Configuration exists at {SETTINGS_PATH}")
+            console.print("[yellow]Use --force to overwrite, or run 'polylogue settings' to modify.")
+            return
+
+    console.print("\n[bold cyan]Welcome to Polylogue![/bold cyan]")
+    console.print("Let's set up your configuration.\n")
+
+    # Step 1: Output directory
+    default_dir = Path.home() / "polylogue-data"
+    console.print(f"[bold]Output Directory[/bold]")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)